packages/devtools-client-adapters/types.js

```diff
@@ -0,0 +1,90 @@
+// @flow
+
+/**
+ * Breakpoint
+ *
+ * @memberof actions/types
+ * @static
+ */
+export type Breakpoint = {
+    id: string,
+    location: Location,
+    loading: boolean,
+    disabled: boolean,
+    text: string,
+    condition: ?string,
+};
+
+/**
+ * BreakpointResult
+ *
+ * @memberof actions/types
+ * @static
+ */
+export type BreakpointResult = {
+    ids: string,
```

Comment:
That's a blocker for this PR. :) We can't merge this until we know it's actually working. If it's not applying the types we have no idea if this PR actually works.